erddap/content/datasets.xml

```diff
@@ -6549,15 +6650,16 @@ AZOMP's sampling scheme off Atlantic Canada and the Labrador Sea involves annual
       <destinationName>geographic_area</destinationName>
       <dataType>String</dataType>
       <addAttributes>
-        <att name="ioos_category">Other</att>
+        <att name="ioos_category">Location</att>
       </addAttributes>
     </dataVariable>
     <dataVariable>
       <sourceName>global:station</sourceName>
       <destinationName>station</destinationName>
       <dataType>String</dataType>
       <addAttributes>
-        <att name="ioos_category">Location</att>
+        <att name="ioos_category">Other</att>
```

Comment:
"Identifier" might be a better ioos_category

Reply:
Will update! Thanks!